test/200-bridge-firewalld.bats

```diff
@@ -212,115 +211,165 @@ function setup() {
 }
 
 @test "$fw_driver - port forwarding ipv4 - tcp" {
+    skip "test requires firewalld same-machine port forwarding for non-localhost IP"
```

Comment:
I guess we have a testing cap in the other files as well, there is no 127.0.0.1 port forwarding test at ALL unless I am missing something.

I would assume we could have something like 
```
@test "$fw_driver - port forwarding with localhost - tcp" {
    test_port_fw hostip="127.0.0.1"
}
```

That should work with all drivers I guess